src/python/py_generate_pipeline.cpp

```diff
@@ -10,6 +10,9 @@
 #include "openvino/genai/llm_pipeline.hpp"
 #include <openvino/runtime/auto/properties.hpp>
 #include "../cpp/src/tokenizers_path.hpp"
+#include "openvino/genai/whisper_generation_config.hpp"
+#include "openvino/genai/whisper_speech_recognition_pipeline.hpp"
+#include "openvino/genai/whisper_speech_recognition_pipeline.hpp"
```

Comment:
maybe we can create a dedicate file for whisper? 
current file can be renamed to LLM pipeline or similar.